src/dmd/utils.d

```diff
@@ -132,3 +132,55 @@ extern (C++) void escapePath(OutBuffer* buf, const(char)* fname)
         fname++;
     }
 }
+
+
+/**
+ * Takes a path, and escapes '(', ')', backslashes and whitespace
+ *
+ * Params:
+ *   buf = Buffer to write the escaped path to
+ *   fname = Path to escape
+ */
+void escapePathSpaces(OutBuffer* buf, const(char)* fname)
```

Comment:
Why is this not used for any other filename arguments?

Reply:
All the other places where `escapePath` was used are in either human-readable or paren-delimited custom format contexts, so there was no need to use this before (as far as I know).